scripts/consolidate-changelogs.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const fs = require('fs');
+
+const directoryPath = path.resolve(__dirname, '../changelogs');
+const changelogPath = `${path.resolve(__dirname, '../')}/CHANGELOG.md`;
+const tempLogsPath = `${path.resolve(__dirname, '../')}/tempLogs.json`;
```

Comment:
It bugs me that this script has an unseen dependency on the `compile-changelogs.js` script.  I mean, it should work, but only as long as you call these scripts in exactly the right order.  For example, looking at the `github_tag` Circle job, it looks like we run `node scripts/release-tag.js` alone.  `scripts/release-tag.js` is actually dependent on that `tempLogs.json` file, but it won't exist in that workflow, because we don't call `scripts/compile-changelogs.js` first.  So that one will fail.  

In general, I like the direction this PR is headed (breaking apart the monolothic scripts into composable parts). But I think the way it would make more sense to me if we avoid using a temp file to pass data between the scripts.  It seems like `consolidate-changelogs.js` should just do its own changelog file parsing (or call `compile-changelogs.js`, where `compile-changelogs.js` returns the data instead of writing to a temp file).  Then, `release-branch.js` and `release-tag.js` should just parse the `CHANGELOG.md` file to grab the changelog data for the release (it should already know the version # to look for).

Reply:
So I started with exporting those data in the tempLogs.json, to share between jobs. If I don't create the temp file e.g. changelogs at the time of running release-tag will be removed already so the compiled changelogs will have no content. The only solution I could think of was to create a file and cache it and remove it after the release-tag job gets the data. Is there another I could do this?

Reply:
e.g. https://github.com/massgov/mayflower/pull/904 This was when I used imports/exports